tests/defaults.yml

```
@@ -2,10 +2,12 @@ parameters:
   kapitan:
     dependencies:
       - type: https
-        source: https://raw.githubusercontent.com/projectsyn/component-resource-locker/v2.1.0/lib/resource-locker.libjsonnet
-        output_path: vendor/lib/resource-locker.libjsonnet
+        source: https://raw.githubusercontent.com/projectsyn/component-patch-operator/v1.0.0/lib/patch-operator.libsonnet
```

Not sure if I missed this in the other PR(s), but you should probably use v1.2.0 here

```suggestion
        source: https://raw.githubusercontent.com/projectsyn/component-patch-operator/v1.2.0/lib/patch-operator.libsonnet
```